osf_tests/test_storage_usage_limits.py

```diff
@@ -22,39 +22,43 @@ def test_limit_default(self, node):
         assert node.storage_limit_status == StorageLimits.DEFAULT
 
     def test_storage_limits(self, node):
+        GBs = 1024 ** 3.0
+
         key = cache_settings.STORAGE_USAGE_KEY.format(target_id=node._id)
-        storage_usage_cache.set(key, int(StorageLimits.OVER_PUBLIC * STORAGE_WARNING_THRESHOLD))
+        storage_usage_cache.set(key, int(STORAGE_LIMIT_PUBLIC * STORAGE_WARNING_THRESHOLD * GBs))
 
-        assert node.storage_limit_status == StorageLimits.APPROACHING_PUBLIC
+        # Compare by name because values are different
+        assert node.storage_limit_status.name == StorageLimits.APPROACHING_PUBLIC.name
```

Comment:
Nit: Once you've moved away from creating custom enums, identify is the preferred comparison mode for Enums (i.e. node.storage_limit_status is StorageLimits.APPROACHING_PUBLIC)